megatron/core/transformer/utils.py

```diff
@@ -29,13 +29,13 @@ def get_linear_layer(rows, columns, init_method, perform_initialization=True):
     return layer
 
 
-@lru_cache(maxsize=32)
+#@lru_cache(maxsize=32)
```

Comment:
Rollback? Or resharging won't work with lru_cache?

There's a hacky way to clear the cache which I use in rl_utils: https://github.com/tdene/Megatron-LM/blob/f3bb645b7d47f916a75b069f60b15211ceb9f096/megatron/rl/rl_utils.py#L2268